lib/runtime_tools/test/system_information_SUITE_data/information_test_report.dat

```diff
@@ -3689,7 +3689,7 @@
                  {application,
                   {otp_mibs,
                    [{description,
-                     "SNMP managment information base for Erlang/OTP nodes."},
+                     "SNMP management information base for Erlang/OTP nodes."},
```

Comment:
Please keep this as it was, it is a dump of an old state of the SNMP application and should probably keep its old flaws.